Autopilot/AttitudeManager/FixedWing.c

```diff
@@ -0,0 +1,248 @@
+/*
+ * File:   FixedWing.c
+ * Author: Chris Hajduk
+ *
+ * Created on July 2, 2015, 8:04 PM
+ */
+
+#include "PWM.h"
+#include "AttitudeManager.h"
+#include "delay.h"
+#include "FixedWing.h"
+#include "ProgramStatus.h"
+
+#if VEHICLE_TYPE == FIXED_WING
```

Comment:
I think you should remove this. Instead the files that require it can either choose to include or exclude the FixedWing.h file based on the Vehicle type property